Improve Account Switching: Support doc updates (#1482)
dnsimple/dnsimple-support: content/articles/restoring-domain.md

##### To restore a domain

> 1.  Log in to DNSimple with your user credentials.

Reviewer: This seems like the kind of thing we could remove from _all_ of our instructions. 😅

```suggestion
```

> 1.  Select the relevant account from the account switcher in the top-right corner of the page.

Reviewer 2: ```suggestion
1.  Select the relevant account from the account switcher at the top-right corner of the screen.
```
